### PR TITLE
Update portuguese translation

### PR DESCRIPTION
**Pull request by KelseyJr**
Added new strings fom eng.ini.

### CODE REVIEW

File: resources/lang/por.ini [hunk 1]
```diff
@@ -3,22 +3,100 @@ language.name = "Portuguese"
...
+flags.hollowclosed = "Oco com extremidades fechadas"
+flags.natural = "Natural"
+; tools
+; wand tool
+tool.wand = "Varinha"
+tool.wand.lore.1 = "Clique com o botão esquerdo do mouse em um bloco para definir a posição 1 de uma seleção"
+tool.wand.lore.2 = "Clique com o botão direito do mouse em um bloco para definir a posição 2 de uma seleção"
+tool.wand.lore.3 = "Use //togglewand para alternar sua funcionalidade"
+tool.wand.disabled = "A ferramenta da varinha está desativada. Use //togglewand para reativá-lo"
+tool.wand.setenabled = "The wand tool is now {%0}!"
```
Comment (inxomnyaa) [inline resources/lang/por.ini]: Seems to be missing

File: resources/lang/por.ini [hunk 2]
```diff
@@ -3,22 +3,100 @@ language.name = "Portuguese"
...
+flags.hollow = "Oco"
+flags.hollowclosed = "Oco com extremidades fechadas"
+flags.natural = "Natural"
+; tools
+; wand tool
+tool.wand = "Varinha"
+tool.wand.lore.1 = "Clique com o botão esquerdo do mouse em um bloco para definir a posição 1 de uma seleção"
+tool.wand.lore.2 = "Clique com o botão direito do mouse em um bloco para definir a posição 2 de uma seleção"
+tool.wand.lore.3 = "Use //togglewand para alternar sua funcionalidade"
+tool.wand.disabled = "A ferramenta da varinha está desativada. Use //togglewand para reativá-lo"
```
Comment (inxomnyaa) [inline resources/lang/por.ini]: > Use

is still here

**Review by inxomnyaa — CHANGES_REQUESTED** [review]

File: resources/lang/por.ini (hunk 1 above)
Comment (inxomnyaa) [reply resources/lang/por.ini]: the {%0} in this line changes into "enabled" or "disabled"

File: resources/lang/por.ini (hunk 2 above)
Comment (KelseyJr) [reply resources/lang/por.ini]: Actually, it is correct. Use means use, in portuguese, but I can translate to other word, if you want.

File: resources/lang/por.ini (hunk 1 above)
Comment (KelseyJr) [reply resources/lang/por.ini]: I'll change it. Sorry for my mistake.

**Comment by KelseyJr** [comment]
I've changed the word Use and translate _tool.wand.setenabled_

**Review by inxomnyaa — APPROVED** [review]

File: resources/lang/por.ini (hunk 2 above)
Comment (inxomnyaa) [reply resources/lang/por.ini]: Intersting - one never stops learning i guess, thanks for clarifying

**Pull request merged**